dashboards/gitlab/gitlab_dashboard.json

```diff
@@ -19,7 +19,7 @@
             "id": "viz.markdown"
           },
           "rawConfiguration": {
-            "text": "![drawing](https://1000logos.net/wp-content/uploads/2022/02/Gitlab-Logo.png)\n"
+            "text": "![drawing](https://1000logos.net/wp-content/uploads/2022/03/Gitlab-Logo-500x282.jpg)\n"
```

Comment:
Please use the image source - https://newrelic.com/sites/default/files/quickstarts/images/icons/gitlab--logo.svg 
We already have the logo hosted at New Relic.

Reply:
Done